page-optimize.php

```diff
@@ -236,6 +251,19 @@ function page_optimize_remove_concat_base_prefix( $original_fs_path ) {
 	return '/page-optimize-resource-outside-base-path/' . basename( $original_fs_path );
 }
 
+function page_optimize_schedule_cache_cleanup() {
+	$cache_folder = false;
+	if ( defined( 'PAGE_OPTIMIZE_CACHE_DIR' ) && ! empty( PAGE_OPTIMIZE_CACHE_DIR ) ) {
+		$cache_folder = PAGE_OPTIMIZE_CACHE_DIR;
+	}
+	$args = [ $cache_folder ];
+
+	// If caching is on, and job isn't queued for current cache folder
+	if( false !== $cache_folder && false === wp_next_scheduled( PAGE_OPTIMIZE_CRON_CACHE_CLEANUP_JOB, $args ) ) {
```

Comment:
It feels a bit weird or indirect to check `$cache_folder` here because we then we use `$args` which is derived from `$cache_folder`. Maybe we should use:

```php
$cleanup_args = false;
if ( defined( 'PAGE_OPTIMIZE_CACHE_DIR' ) && ! empty( PAGE_OPTIMIZE_CACHE_DIR ) ) {
    $cleanup_args = [ PAGE_OPTIMIZE_CACHE_DIR ];
}

if ( false !== $cleanup_args && false === wp_next_scheduled( PAGE_OPTIMIZE_CRON_CACHE_CLEANUP_JOB, $cleanup_args ) ) {
```

Just thinking aloud. Not super opinionated about this.

Reply:
Me neither. It reads ok to me. If `cache_folder` is false, then cache is disabled, otherwise, try queueing.